scripts/extract_strings.sh

```diff
@@ -103,9 +103,15 @@ else
 fi
 
 # Fluent extraction
+# fxa-payments-server
 cp $PAYMENTS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/templates
+cp $PAYMENTS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/en_US
+# fxa-settings
 cp $SETTINGS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/templates
+cp $SETTINGS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/en_US
+# fxa-auth-server
 cp $MAILER_DIR/public/locales/en/*.ftl $L10N_DIR/locale/templates
+cp $MAILER_DIR/public/locales/en/*.ftl $L10N_DIR/locale/en
 
```

Comment:
I would do the copy at the end once, with a comment explaining why we need to copy files once again.

```
# We need to copy the FTL files over to "en" to ensure users with English
# will always get good fallback content.
cp $L10N_DIR/locale/templates/*.flt $L10N_DIR/locale/en
```

But we also need to check with @mathjazz: this PR would results in pot files being in templates, as expected, but we'll have FTL files in `en`, `en-US`, and `templates`. Can you confirm this will not confuse Pontoon's logic?

Reply:
It will not confuse Pontoon. It takes source files from the `templates` folder. If it doesn't find them there, it checks `en-US` and then `en`.

I wonder if we can store all reference files in one directory (e.g. `en-US`) and take care of the fallbacks programmatically.

Reply:
> I wonder if we can store all reference files in one directory (e.g. en-US) and take care of the fallbacks programmatically.

I assume Pontoon only checks one folder out of the three (templates -> en-US -> en)? If that's correct, we need the .pot files in `templates` for GetText, but if we put the FTL files in `en-US` they won't be considered at all. So, we need all source files in `templates` for the time being (or until GetText is needed).

Could you clarify what you mean with the last part of the sentence?

Reply:
Yes, Pontoon only checks one folder out of three (templates, en-US, en), which is why I'm wondering if that could be the case on the Firefox Accounts side as well.

From Pontoon's perspective there's no need for the `en-US` and `en` folders.

Reply:
Pontoon might not need them, but FxA needs a fallback locale with en-US content (that's the reason for this PR in the first place). 

`templates` would really not be an intuitive choice for that, as we always have either `en` or `en-US` in all existing Fluent projects.

Reply:
> It will not confuse Pontoon. It takes source files from the templates folder. If it doesn't find them there, it checks `en-US` and then `en`.

Nice, I didn't realize it would check the `locale` directory if it couldn't find one from the `templates` directory. If I'm understanding this conversation correctly, it sounds like so long as we need gettext files or so long as gettext files are outputted to `templates`, that we can't remove our FTL files from `templates` in favor of relying on them being inside `locale/en` or `locale/en_US`? But then, when we do move off of gettext, we can potentially stop placing our FTL files into `templates`, so long as all of our FTL files are either in `locale/en` or `locale/en_US`. I don't think we want to alter any gettext files or output at the moment, we only have one package left to phase out before we're entirely on Fluent.

Reply:
Thanks for clarifying. I pushed an update with a couple comments.